api/storage_service.cc

```diff
@@ -428,8 +428,7 @@ void set_repair(http_context& ctx, routes& r, sharded<repair_service>& repair) {
                 continue;
             }
             if (!options.contains(x.first)) {
-                return make_exception_future<json::json_return_type>(
-                        httpd::bad_param_exception(format("option {} is not supported", x.first)));
+                throw httpd::bad_param_exception(format("option {} is not supported", x.first));
```

Comment:
Patch says "coroutinize", but we also change the exception here. Does this hunk belong to the next commit?

Reply:
my bad. will fix it in the next revision.

Reply:
wait, i am not changing it. it was `httpd::bad_param_exception`, and it still is.

Reply:
In the past, I saw weird result when turning code in the api into coroutine. Please double check the code still works with coroutine now.

Reply:
sure, thanks for the reminder. i tested with `test_repair_small_table_optimization` in dtest, it worked as expected.